packages/location-management/src/components/LocationForm/ExtraFields/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { sendErrorNotification } from '@opensrp/notifications';
+import { OpenSRPService } from '@opensrp/react-utils';
+import { LOCATION_UNIT_EXTRA_FIELDS_IDENTIFIER } from '../../../constants';
+import { loadSettings } from '../../../helpers/dataLoaders';
+import React, { useEffect, useState } from 'react';
+import { Setting, validationRules } from '../utils';
+import { Form, Input } from 'antd';
+import { OPENSRP_API_BASE_URL } from '@opensrp/server-service';
+
+const { List, Item: FormItem } = Form;
+
+export interface ExtraFieldProps {
+  baseURL: string;
+  service: typeof OpenSRPService;
+  disabled: boolean;
+  hidden: boolean;
+}
+
+const defaultExtraFieldProps = {
+  baseURL: OPENSRP_API_BASE_URL,
+  service: OpenSRPService,
+  disabled: false,
+  hidden: false,
+};
+
+/** renders the extra fields form fields
+ *
+ * @param props - the components props
+ */
+const ExtraFields = (props: ExtraFieldProps) => {
```

Comment:
What is component used for?

Reply:
This renders the extra adhoc fields, these are fields whose properties are stored in the settings endpoint, 
PS: this pr does not add new functionality it just refactors, re-organizes what was previously there.

Reply:
will update the doc in the code.